test/spec/task/task.spec.ts

```diff
@@ -0,0 +1,198 @@
+        .send({
+          name: `${i}`
+        });
+    }
+
+    // getting a list of tasks
+    const res = await chai
+      .request(appTwo)
+      .get(`/api/accounts/${accountId}/tasks`)
+      .set('content-type', 'application/json')
+      .set('Authorization', `Bearer ${accessToken}`)
+      .send();
+
+    expect(res).to.have.status(200);
+    expect(res.body).to.be.an('array');
+
+    // As we are not mocking DB and the tests are running on real DB URL. 
+    // So there may be more than 2 tasks also in DB.
+    // So we are just making sure that we get an array of tasks for that account using this API URL.
+    
+    expect(res.body.length).to.be.greaterThan(0);
+  });
+
+  it('POST "/api/accounts/:accountId/tasks" should create a new task.', async () => {
+    const params = {
+      name: 'simple task.'
+    };
+
+    const res = await chai
+      .request(appTwo)
+      .post(`/api/accounts/${accountId}/tasks`)
+      .set('content-type', 'application/json')
+      .set('Authorization', `Bearer ${accessToken}`)
+      .send(params);
+
+    expect(res).to.have.status(201);
+    expect(res.body).to.have.property('id');
+    expect(res.body).to.have.property('account');
+    expect(res.body).to.have.property('name');
+    expect(res.body.account).to.eq(accountId);
+    expect(res.body.name).to.eq('simple task.');
+
+    // checking if the task is created in DB
+    const createdTask = await TaskService.getTask({
+      accountId,
+      taskId: res.body.id
+    });
+    expect(createdTask).to.have.property('id');
+    expect(createdTask.id).to.eq(res.body.id);
+  });
+
+  it('GET "/api/accounts/:accountId/tasks/:taskId" should return a particular task.', async () => {
+
+    let res: any;
+    const taskTitle = 'simple task.'
+
+    // creating a task before
+    const params = {
+      name: taskTitle
+    };
+
+    res = await chai
+      .request(appTwo)
+      .post(`/api/accounts/${accountId}/tasks`)
+      .set('content-type', 'application/json')
+      .set('Authorization', `Bearer ${accessToken}`)
+      .send(params);
+    expect(res).to.have.status(200);
+
+    const taskId = res.body.id;
+
+    // getting the task
+    res = await chai
+      .request(appTwo)
+      .get(`/api/accounts/${accountId}/tasks/${taskId}`)
+      .set('content-type', 'application/json')
+      .set('Authorization', `Bearer ${accessToken}`)
+      .send();
+
+    expect(res).to.have.status(200);
+    expect(res.body).to.have.property('id');
+    expect(res.body.name).to.eq(taskTitle);
+    expect(res.body.id).to.eq(taskId);
+
+    // checking if a task is created in DB.
+    const particularTask = await TaskService.getTask({
+      accountId,
+      taskId: res.body.id
+    });
+    expect(particularTask).not.to.be.undefined;
+    expect(particularTask).to.have.property('id');
+  });
+
+  it('DELETE "/api/accounts/:accountId/tasks/:taskId" should change "active" flag of task to be false.', async () => {
+
+    let res: any;
```

Comment:
As pre-condition, we should create a task using TaskService and assert its in database.

Spec should delete the task.

As post condition, it should assert fetching from API return 404